crates/nu-command/src/dataframe/lazy/fill_nan.rs

```diff
@@ -0,0 +1,112 @@
+use crate::dataframe::values::{Column, NuDataFrame, NuExpression, NuLazyFrame};
+use nu_engine::CallExt;
+use nu_protocol::{
+    ast::Call,
+    engine::{Command, EngineState, Stack},
+    Category, Example, PipelineData, ShellError, Signature, SyntaxShape, Type, Value,
+};
+use polars::datatypes::DataType;
+
+#[derive(Clone)]
+pub struct LazyFillNA;
+
+impl Command for LazyFillNA {
+    fn name(&self) -> &str {
+        "fill-nan"
+    }
+
+    fn usage(&self) -> &str {
+        "Replaces NA values with the given expression"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build(self.name())
+            .required(
+                "fill",
+                SyntaxShape::Any,
+                "Expression to use to fill the NAN values",
+            )
+            .input_type(Type::Custom("dataframe".into()))
+            .output_type(Type::Custom("dataframe".into()))
+            .category(Category::Custom("lazyframe".into()))
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "Fills the NaN values by 0",
+            example: "[1 2 2 3 3] | into df | fill-nan 0",
```

Comment:
This example is not really helpful as it doesn't show a value being replaced.

Reply:
Fixed